### PR TITLE
Rviz Utils Namespace & Tests

### PR DESCRIPTION
**Pull request by RobynCastro**

### CODE REVIEW

File: src/sb_utils/CMakeLists.txt [hunk 1]
```diff
@@ -60,7 +60,11 @@ target_link_libraries(sb_utils ${catkin_LIBRARIES})
...
-#   target_link_libraries(${PROJECT_NAME}-test ${PROJECT_NAME})
-# endif()
+if (CATKIN_ENABLE_TESTING)
+    # Adding rostest to the package
+    find_package(rostest REQUIRED)
+    # name the test and link it to the .test file and the .cpp file itself, this will allow
+    # "catkin_make run_tests" to be able to find and run this rostest
+    add_rostest_gtest(rviz_utils_rostest test/rviz_utils_test.test test/rviz-utils-rostest.cpp src/RvizUtils.cpp)
+    target_link_libraries(rviz_utils_rostest ${catkin_LIBRARIES})
+endif()
```
Comment (garethellis0) [inline src/sb_utils/CMakeLists.txt]: :+1: for tests!

File: src/sb_utils/include/RvizUtils.h [hunk 2]
```diff
@@ -41,7 +41,8 @@ class RvizUtils {
      *
      *  @return an rviz marker
      */
-    static visualization_msgs::Marker
+    visualization_msgs::Marker
+
```
Comment (garethellis0) [inline src/sb_utils/include/RvizUtils.h]: Extra space here?

File: src/sb_utils/test/rviz-utils-rostest.cpp [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+    EXPECT_TRUE(frame_id == test_marker.header.frame_id);
+    EXPECT_TRUE(ns == test_marker.ns);
+}
+
+int main(int argc, char** argv) {
+    ros::init(argc, argv, "rviz_utils_test");
+    ros::start();
+    testing::InitGoogleTest(&argc, argv);
+    return RUN_ALL_TESTS();
+}
```
Comment (garethellis0) [inline src/sb_utils/test/rviz-utils-rostest.cpp]: Good job with these, they're clean, readable, and seem fairly comprehensive.

File: src/sb_utils/test/rviz-utils-rostest.cpp [hunk 4]
```diff
@@ -0,0 +1,276 @@
+/*
```
Comment (garethellis0) [inline src/sb_utils/test/rviz-utils-rostest.cpp]: Just for naming convention's sake, this should be `rviz_utils_rostest.cpp`

Comment (RobynCastro) [reply src/sb_utils/test/rviz-utils-rostest.cpp]: !fixed

**Pull request merged**